examples/signal/plot_covariance_estimation.py

```diff
@@ -0,0 +1,205 @@
+"""
+===============================================================================
+Estimate covariance with different time windows
+===============================================================================
+
+Covariance estimators comparison for different EEG signal lengths and their
+impact on classification [1]_.
+"""
+# Author: Sylvain Chevallier
+#
+# License: BSD (3-clause)
+
+import numpy as np
+import pandas as pd
+import seaborn as sns
```

Comment:
`seaborn` should be imported after `matplotlib`, to avoid the multiple warnings:

> UserWarning: Trying to register the cmap * which already exists.